src/components/victory-axis.jsx

```diff
@@ -47,9 +47,17 @@ const styles = {
   }
 };
 
-class VAxis extends React.Component {
+@Radium
+export default class VictoryAxis extends React.Component {
+  static role = "axis";
```

Comment:
For this I was thinking `role: "axis"`, which is equivalent to `VAxis.prototype.role = "axis"` rather than `VAxis.role = "axis"`. If I understand how this will be used, we'll be looping over child instances – I think being able to access it like `child.role` looks better than having to access it on the constructor class – plus, complex components in the future could maybe dynamically set their `role`.